frontend/packages/system/src/NotificationEvents/DetailView/DetailView.tsx

```diff
@@ -0,0 +1,130 @@
+        {/* if we have a config_id, create a link to edit the config */}
+        {entity?.notificationConfigId && (
+          <LoadingButton
+            isLoading={isLoading}
+            onClick={() => {
+              navigate(
+                configRoute(
+                  entity.notificationConfig?.kind ?? ConfigKind.Scheduled,
+                  entity.notificationConfigId ?? ''
+                )
+              );
+            }}
+            variant="outlined"
+            endIcon={<EditIcon />}
+          >
+            {t('button.edit-config')} ({entity.notificationConfig?.title})
+          </LoadingButton>
+        )}
+      </AppBarButtonsPortal>
+      <AppBarContentPortal sx={{ paddingBottom: '16px', flex: 1 }}>
+        <Box display="flex" flexDirection="row" gap={1} width="100%">
+          <Box gap={1} alignItems="flex" display="flex" flex={1}>
+            <Box flex={1} display="flex">
+              <Box flex={0.2} gap={1}>
+                <Stack gap={1}>
+                  <NotificationStatusChip
+                    status={entity?.status ?? EventStatus.Errored}
+                  />
+                  <RelativeTimeDate d={entity?.sentAt} />
+                </Stack>
+              </Box>
+              <Box flex={1} justifyContent="center" display="flex" gap={1}>
+                {entity?.errorMessage ? (
+                  <TextArea value={entity?.errorMessage} />
+                ) : (
+                  <Typography variant="body1">No Error</Typography>
+                )}
+              </Box>
+              <Box flex={1} justifyContent="right" display="flex" gap={1}>
+                <Stack gap={1}>
+                  <Typography variant="body1">
+                    Created:
+                    <RelativeTimeDate d={entity?.createdAt} />
+                  </Typography>
+                  <Typography variant="body1">
+                    Updated:
+                    <RelativeTimeDate d={entity?.updatedAt} />
+                  </Typography>
+                </Stack>
+              </Box>
+            </Box>
+          </Box>
+        </Box>
+      </AppBarContentPortal>
+      {/* Description/Details section */}
+      <Box sx={{ width: '100%', display: 'flex', flexDirection: 'column' }}>
+        <Box sx={{ flex: '1', overflow: 'auto', padding: '4px' }}>
+          {isLoading ? (
+            <BasicSpinner />
+          ) : (
+            <>
+              <Typography variant="h4">{entity?.title}</Typography>
+              <Typography variant="h6">
+                {entity?.toAddress} ({entity?.notificationType})
+              </Typography>
+              <TextArea
+                minRows={2}
+                maxRows={25}
+                sx={{
+                  border: '1px solid',
+                  borderColor: 'grey.100',
+                  width: '100%',
+                }}
+                value={entity?.message}
+              />
+            </>
+          )}
+        </Box>
+      </Box>
```

Comment:
There's a couple of `Box` here as well, and they don't add anything. Also needs a bit of a pad
```suggestion
<Box
        sx={{ width: '100%' }}
        padding={2}
        display="flex"
        flexDirection="column"
      >
        {isLoading ? (
          <BasicSpinner />
        ) : (
          <>
            <Typography variant="h4">{entity?.title}</Typography>
            <Typography variant="h6">
              {entity?.toAddress} ({entity?.notificationType})
            </Typography>
            <TextArea
              minRows={2}
              maxRows={25}
              sx={{
                border: '1px solid',
                borderColor: 'grey.100',
                width: '100%',
              }}
              value={entity?.message}
            />
          </>
        )}
      </Box>
```